lib/ui/recommendation.js

```diff
@@ -6,101 +6,12 @@ const {classes: Cc, utils: Cu} = Components;
 
 Cu.import('resource://gre/modules/XPCOMUtils.jsm');
 XPCOMUtils.defineLazyModuleGetter(this, 'Services',
- 'resource://gre/modules/Services.jsm');
+  'resource://gre/modules/Services.jsm');
+XPCOMUtils.defineLazyModuleGetter(this, 'RecommendationRow',
+  'chrome://universalsearch-lib/content/ui/recommendation-row.js');
```

Comment:
Nope, I meant it for both cases - let's just use constructor injection to hand things to individual modules